types/ember__routing/test/router.ts

```diff
@@ -58,9 +59,31 @@ const RouterServiceConsumer = Service.extend({
         router
             .on('routeWillChange', transition => {
                 const to = transition.to;
+                assertType<RouteInfo | null>(to.child);
```

Comment:
Can we change these to use [dtslint](https://github.com/Microsoft/dtslint) tests? assertType can let inappropriately wide types slip through (i.e., if `to.child` is of type `any`, this assertion will pass).

Example
```ts
to.child; // $ExpectType RouteInfo | null
```

Reply:
Sure thing. Didn't know that was an option.